virtualization/windowscontainers/container-networking/architecture.md

```diff
@@ -55,13 +48,24 @@ The Host Networking Service (HNS) and the Host Compute Service (HCS) work togeth
   - HNS creates NAT and IP pools as required
 
 ### Endpoint Creation
-  - HCS creates network compartment per container endpoint
-  - HNS/HCS places v(m)NIC inside network compartment
+  - HNS creates network namespace per container endpoint
+  - HNS/HCS places v(m)NIC inside network namespace
   - HNS creates (vSwitch) ports
   - HNS assigns IP address, DNS information, routes, etc. (subject to networking mode) endpoint
 
 ### Policy Creation
-  - HNS creates WinNAT port forwarding rules / mappings with corresponding Windows Firewall ALLOW rules
+  - Default NAT network: HNS creates WinNAT port forwarding rules / mappings with corresponding Windows Firewall ALLOW rules
+  - All other networks: HNS utilizes the Virtual Filtering Platform (VFP) for policy creation
+    - This includes: load balancing, ACLs, encapsulation, etc.
```

Comment:
Add another sub-bullet after "This includes: load balancing, ACLs, …"
"- Look for our HNS APIs and schema to be published soon"